src/Kernel32.Shared/NTStatusException.cs

```diff
@@ -65,6 +65,32 @@ protected NTStatusException(SerializationInfo info, StreamingContext context)
         /// </summary>
         public NTSTATUS NativeErrorCode { get; }
 
+        /// <summary>
+        /// Creates a new instance of the <see cref="NTStatusException"/> class with localized message.
+        /// </summary>
+        /// <param name="statusCode">The status code identifying the error.</param>
+        /// <param name="localize">Indicates wether the message should be localized</param>
+        /// <returns>The localized NTStatusException</returns>
+        public static NTStatusException Create(NTSTATUS statusCode, bool localize)
```

Comment:
I was trying to to break what we already have. May I alter the default constructor behavior?